lab/changelog.md

```diff
@@ -4,6 +4,17 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.3.1 - Nov 4, 2022
+
+### Bug Fixes
+
+* Fix handling single-line commented Cypher code in query execution.
+* Fix the map toggle freeze in the graph view.
+* Fix showing multiple maps in the rich collections.
+* Fix missing node and edge information on graph schema.
+* Fix the query reordering in the rich collections.
+* Fix using a previous description when adding a new query to the rich collection.
```

Comment:
I would like us to start writing user-oriented release notes/changelogs... 
The shift would be from What we did to What you can now do/or not have problems with any more
For example, as a user I have no idea what half of these fixes actually do for me... or what problems they fix from my perspective

For example: 
* Writing a single-line comment in the Cypher code no longer results in an error
* "I have no idea what this actually means"
* "as well"
* All the information about nodes and edges on the graph schema is now previewed as expected. 
* "I have no idea what happened before and what this means now for me"
* A bug that would not reset the description field when adding a new query to the query collection has been fixed. 


```suggestion
* Fix handling single-line commented Cypher code in query execution.
* Fix the map toggle freeze in the graph view.
* Fix showing multiple maps in the rich collections.
* Fix missing node and edge information on graph schema.
* Fix the query reordering in the rich collections.
* Fix using a previous description when adding a new query to the rich collection.
```

Reply:
Good feedback! It definitely makes sense.

+ thanks a lot for few examples, 1st, 4th and 6th lines are super well written :) 

Will update the notes.

Reply:
I just added an update, I am not really happy with the 5th one - I think it is still complex. What do you think?

Btw I also added the missing one (the last one).